src/NUnitFramework/framework/Constraints/AllItemsConstraint.cs

```diff
@@ -48,7 +48,9 @@ public AllItemsConstraint(IConstraint itemConstraint)
         /// Apply the item constraint to each item in the collection,
         /// failing if any item fails.
         /// </summary>
-        /// <param name="actual"></param>
+        /// <typeparam name="TActual"></typeparam>
```

Comment:
Why add this type paramter without any text? I can't see what value it would add in any case.